### ISSUE TITLE
GeoTax losing specialPurposeDistrictsTax?

### ISSUE THREAD

**Issue by pdeffendol** [PitneyBowes/LocationIntelligenceSDK-CSharp#1]
I'm getting a different response from the GeoTax C# SDK than from a direct call to the REST API for a particular address.  Specifically, this address has two special purpose tax districts, but the sales tax structure is coming back "null".

Here is the JSON response, obtained from the "Try It" page on the website.  The address submitted is "4053 Dawn Ct, Boulder, CO 80304".

```
{
    "confidence": 100,
    "jurisdiction": {
        "state": {
            "code": "08",
            "name": "CO"
        },
        "county": {
            "code": "013",
            "name": "Boulder"
        },
        "place": {
            "name": "Boulder",
            "code": "07850",
            "classCode": "C1",
            "incorporatedFlag": "Inc",
            "lastAnnexedDate": "10/2011",
            "lastUpdatedDate": "06/2016",
            "lastVerifiedDate": "09/2015"
        },
        "spds": [
            {
                "districtName": "Scientific and Cultural Facilities Dist",
                "districtCode": "SCD",
                "districtNumber": "00004",
                "versionDate": "092006",
                "effectiveDate": "012004",
                "compiledDate": "092006",
                "updateDate": "062016"
            },
            {
                "districtName": "The Regional Transportation District",
                "districtCode": "RTA",
                "districtNumber": "00008",
                "versionDate": "112006",
                "effectiveDate": "012005",
                "compiledDate": "092006",
                "updateDate": "092016"
            }
        ]
    },
    "matchedAddress": {
        "formattedAddress": "4053 Dawn Ct Boulder, CO  80304-0984",
        "mainAddressLine": "4053 Dawn Ct",
        "addressLastLine": "Boulder, CO  80304-0984",
        "placeName": "",
        "areaName1": "CO",
        "areaName2": "",
        "areaName3": "Boulder",
        "areaName4": "",
        "postCode": "80304",
        "postCodeExt": "0984",
        "country": "USA",
        "addressNumber": "",
        "streetName": "",
        "unitType": "",
        "unitValue": ""
    },
    "salesTax": {
        "totalTaxRate": 0.08845,
        "stateTaxRate": 0.029,
        "countyTaxRate": 0.00985,
        "municipalTaxRate": 0.0386,
        "spdsTax": [
            {
                "districtNumber": "00004",
                "taxRate": 0.001
            },
            {
                "districtNumber": "00008",
                "taxRate": 0.01
            }
        ]
    },
    "useTax": {
        "totalTaxRate": 0.0786,
        "stateTaxRate": 0.029,
        "countyTaxRate": 0,
        "municipalTaxRate": 0.0386,
        "spdsTax": [
            {
                "districtNumber": "00004",
                "taxRate": 0.001
            },
            {
                "districtNumber": "00008",
                "taxRate": 0.01
            }
        ]
    }
}
```

When I use the C# SDK and inspect the TaxRateResponse, the value for salesTax.specialPurposeDistrictsTax is "null" rather that a 2-item array.

![watch](https://cloud.githubusercontent.com/assets/8208/20817878/78ff0776-b7e7-11e6-86e0-998eb6eb1296.png)

**Comment by pdeffendol** [comment]
Looks fixed in bbe0141 - I'll run off my own compiled version until a new release is out.  Thanks!

**Comment by ghost** [comment]
Hi Pete,

The issue is fixed and is available in CSharp SDK Git codebase. The compiled CSharp SDK will be available  for download from Pitney Bowes Location Intelligence site on Monday 5th Dec 2016.

Regards
Amit
